R/rdeephaven/lib/build-cpp.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+export SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+export CLIENT=$SCRIPT_DIR/cpp-client
+
+# First, we build all C++ dependencies
+# cd cpp-dependencies
+# chmod +x ./build-dependencies.sh
+# ./build-dependencies.sh
+# cd $SCRIPT_DIR
```

Comment:
Why commented out?

Reply:
Commented out so I could rebuild the client without rebuilding the dependencies. I should add that as a flag argument